package.json

```diff
@@ -6,7 +6,7 @@
   "private": true,
   "scripts": {
     "a11y": "yarn workspace client lighthouse",
-    "dev": "export $(cat config/dev.env | xargs); npm-run-all --parallel dev:*",
```

Comment:
this data is actually needed for the server, not the client, for what it's worth. this webpack file is for the client? I'm not sure how yarn does this under the hood, but it's probably the same env for both the client and serve process.

Reply:
It is already loaded by the server in `yarn workspace server dev` by `dotenv` this was just overwriting it with the wrong variables.